### PR TITLE
DOC-9895: Add FTS changes to What's New document

### PR DESCRIPTION
**Pull request by RayOffiah**

### CODE REVIEW

**Review by tonyjhillman — APPROVED** [review]

File: modules/introduction/pages/whats-new.adoc [hunk 1]
```diff
@@ -93,22 +93,33 @@ Scope Hierarchy Support::
...
 
 Improved Search in the UI::
 The FTS UI will now accept complex full text search terms as well as basic search strings.
 
+File-Based Rebalance in FTS::
+Prior to version{nbsp}7.1, FTS index partitions were rebuilt from scratch on each node during rebalance operations. 
+Under version{nbsp}7.1, the index partition files are copied to the nodes, allowing for faster rebalancing.
+
+Increase in the Number of Full-Text Indexes Supported::
+FTS now supports up 750 indexes per node.
```
Comment (sreekanth-cb) [inline modules/introduction/pages/whats-new.adoc]: FTS can now support up to 500 indexes per cluster. 
(even though the 750 is the internal number we aren't planning to advertise it)

File: modules/introduction/pages/whats-new.adoc [hunk 2]
```diff
@@ -93,22 +93,33 @@ Scope Hierarchy Support::
...
+
 === Index Service Enhancements
 
 The Index Service now provides:
 
-* _Smart Batching_, which allows the rebuilding of indexes, during rebalance, to be maintained at an optimal performance-level.
+Smart Batching::
```
Comment (sreekanth-cb) [inline modules/introduction/pages/whats-new.adoc:119]: These belong to GSI .. and don't think it has to come under FTS changes?

File: modules/introduction/pages/whats-new.adoc (hunk 1 above)
Comment (abhinavdangeti) [reply modules/introduction/pages/whats-new.adoc]: Thought we were advertising 500, no?

File: modules/introduction/pages/whats-new.adoc (hunk 2 above)
Comment (abhinavdangeti) [reply modules/introduction/pages/whats-new.adoc:119]: Ya, not applicable to FTS.

File: modules/introduction/pages/whats-new.adoc [hunk 3]
```diff
@@ -93,22 +93,33 @@ Scope Hierarchy Support::
...
 The Index Service now provides:
 
-* _Smart Batching_, which allows the rebuilding of indexes, during rebalance, to be maintained at an optimal performance-level.
+Smart Batching:: 
+Allows the rebuilding of indexes, during rebalance, to be maintained at an optimal performance-level.
 For information, see xref:learn:clusters-and-availability/rebalance.adoc#smart-batching[Smart Batching].
 
-* _Array Flattening_, whereby the elements of arrays can be more efficiently indexed.
+Array Flattening::
+Whereby the elements of arrays can be more efficiently indexed.
```
Comment (abhinavdangeti) [inline modules/introduction/pages/whats-new.adoc:124]: We didn't do anything new here.

**Review by abhinavdangeti — CHANGES_REQUESTED** [review]

**Review by sreekanth-cb — COMMENTED** [review]
.

File: modules/introduction/pages/whats-new.adoc (hunk 2 above)
Comment (RayOffiah) [reply modules/introduction/pages/whats-new.adoc:119]: Don't worry about those; they're not in your section. I just reformatted them while I was there.

Sorry for the confusion.

File: modules/introduction/pages/whats-new.adoc (hunk 3 above)
Comment (RayOffiah) [reply modules/introduction/pages/whats-new.adoc:124]: Yes, that's not your section.  Don't worry.

I should have done the reformatting under a separate PR

File: modules/introduction/pages/whats-new.adoc (hunk 1 above)
Comment (RayOffiah) [reply modules/introduction/pages/whats-new.adoc]: Fixed

File: modules/introduction/pages/whats-new.adoc [hunk 4]
```diff
@@ -93,22 +93,33 @@ Scope Hierarchy Support::
...
 
 Improved Search in the UI::
 The FTS UI will now accept complex full text search terms as well as basic search strings.
 
+File-Based Rebalance in FTS::
+Prior to version{nbsp}7.1, FTS index partitions were rebuilt from scratch on each node during rebalance operations. 
+Under version{nbsp}7.1, the index partition files are copied to the nodes, allowing for faster rebalancing.
+
+Increase in the Number of Full-Text Indexes Supported::
+FTS now supports up 500 indexes per cluster.
```
Comment (abhinavdangeti) [inline modules/introduction/pages/whats-new.adoc]: Worth mentioning 500 index partitions as opposed to 500 indexes.

File: modules/introduction/pages/whats-new.adoc [hunk 5]
```diff
@@ -93,22 +93,33 @@ Scope Hierarchy Support::
...
 
 Improved Search in the UI::
 The FTS UI will now accept complex full text search terms as well as basic search strings.
 
+File-Based Rebalance in FTS::
+Prior to version{nbsp}7.1, FTS index partitions were rebuilt from scratch on each node during rebalance operations. 
+Under version{nbsp}7.1, the index partition files are copied to the nodes, allowing for faster rebalancing.
+
+Increase in the Number of Full-Text Indexes Supported::
+FTS now supports up 500 index partitions per cluster.
```
Comment (abhinavdangeti) [inline modules/introduction/pages/whats-new.adoc]: up to*

**Review by abhinavdangeti — APPROVED** [review]

File: modules/introduction/pages/whats-new.adoc (hunk 5 above)
Comment (RayOffiah) [reply modules/introduction/pages/whats-new.adoc]: Good spot :-)

Comment (RayOffiah) [reply modules/introduction/pages/whats-new.adoc]: Fixed

**Pull request merged**